src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -0,0 +1,1075 @@
+package org.prebid.server.functional.tests
+
+import org.prebid.server.functional.model.request.auction.Audio
+import org.prebid.server.functional.model.request.auction.BidRequest
+import org.prebid.server.functional.model.request.auction.Content
+import org.prebid.server.functional.model.request.auction.Device
+import org.prebid.server.functional.model.request.auction.Eid
+import org.prebid.server.functional.model.request.auction.Network
+import org.prebid.server.functional.model.request.auction.Producer
+import org.prebid.server.functional.model.request.auction.Publisher
+import org.prebid.server.functional.model.request.auction.Regs
+import org.prebid.server.functional.model.request.auction.Source
+import org.prebid.server.functional.model.request.auction.User
+import org.prebid.server.functional.model.request.auction.UserAgent
+import org.prebid.server.functional.model.request.auction.Video
+import org.prebid.server.functional.model.response.auction.BidResponse
+import org.prebid.server.functional.service.PrebidServerService
+import org.prebid.server.functional.util.PBSUtils
+import spock.lang.Shared
+
+import static org.prebid.server.functional.model.request.auction.Content.Channel
+import static org.prebid.server.functional.model.request.auction.DistributionChannel.APP
+
+class OrtbConverterSpec extends BaseSpec {
+
+    @Shared
+    PrebidServerService prebidServerService = pbsServiceFactory.getService(["adapters.generic.ortb-version": "2.6"])
```

Comment:
I'd just plain spawned both 2.5 and 2.6 instances here and use them. Because using default one will drift next time we add new OpenRTB version.